### ISSUE TITLE
object-type-curly-spacing should not trim multi-line expressions

### ISSUE THREAD

**Issue by gajus** [gajus/eslint-plugin-flowtype#480]
Input:

```js
const slackUserSnackTimeEventInvitation: {|
  +id: DatabaseRecordIdType,
  +snackTimeEventId: DatabaseRecordIdType,
  +slackUserId: DatabaseConnectionType,
  +timezoneName: string,
|} = await connection.one(sql`

```

becomes:

```js
const slackUserSnackTimeEventInvitation: {|
  +id: DatabaseRecordIdType,
  +snackTimeEventId: DatabaseRecordIdType,
  +slackUserId: DatabaseConnectionType,
  +timezoneName: string,|} = await connection.one(sql`

```

This seems undesirable.

**Comment by gajus** [comment]
@angelica-bocanegra

**Comment by angelica-bocanegra** [comment]
Ah okay I will fix this, looking at eslint's implementation of `object-type-curly-spacing` the rule is triggered only when the token and curly brace is on the same line. I will add a check see when it is on the same line or not 👍

**Comment by gajus** [comment]
Thank you
